CrazyCanvas/Include/Match/MatchClient.h

```diff
@@ -32,9 +32,10 @@ class MatchClient : public MatchBase
 	bool OnPacketGameOverReceived(const PacketReceivedEvent<PacketGameOver>& event);
 
 private:
-	bool m_ClientSideBegun = false;
+	bool m_ClientSideBegun	= false;
```

Comment:
Why?

Reply:
why what?

Reply:
For some reaason I have tabbed, must've been a mistake or something because I haven't touched this variable outside of MatchClient

Reply:
As you can see from the red and the green line, you have added one extra unnecessary space character.
Why is this a change?